core/src/main/scala/filodb.core/binaryrecord2/RecordSchema.scala

```diff
@@ -288,6 +299,12 @@ class StringifyMapItemConsumer extends MapItemConsumer {
   }
 }
 
+class MapifyMapItemConsumer(toAdd: collection.mutable.Map[String, String]) extends MapItemConsumer {
```

Comment:
You could just use the output of `StringifyMapItemConsumer` and do .toMap....  less code duplication